### PR TITLE
Transcript scoller/emma

### PR DESCRIPTION
**Pull request by EmmaHkw**
Added a scroller component to the upload page (it will not be on the upload or landing page in the final product, it is just there for now for testing purposes).

Added transcript-scroller.component.css and transcript-scroller.component.js. Modified uploadTranscript.js for testing purposes.

### CODE REVIEW

File: mern/client/src/Components/transcript-scroller.component.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+.scroller { 
+  background-color: rgb(221, 221, 221);
+  height: 700px;
+  width: 500px;
+  border-radius: 20px;
+  overflow-y: auto;
+  overflow-x: auto;
+ }
+
+@font-face {
```
Comment (anishalatchman) [inline mern/client/src/Components/transcript-scroller.component.css:30]: shoudln't need to import the font in your css file. having it in tailwind.config should be enough

File: mern/client/src/screens/Landing/uploadTranscript.js [hunk 2]
```diff
@@ -38,6 +41,9 @@ function UploadTranscript() {
           text={"Go Back"}
         />
       </div>
+      <div class="scroller">
```
Comment (anishalatchman) [inline mern/client/src/screens/Landing/uploadTranscript.js:44]: maybe make a new flow page to test the scroller independently? then we can build the new flow page off of that.

**Review by anishalatchman — CHANGES_REQUESTED** [review]
this branch doesn't have the recoverSession page. i also noticed some styling differences that i wouldn't want to merge (the headings are smaller and the nav bar is italicised). 

can we pull from integration again before merging?

**Review by EmmaHkw — COMMENTED** [review]
Integrated new changes from integration to prevent any conflicts

File: mern/client/src/Components/transcript-scroller.component.css (hunk 1 above)
Comment (EmmaHkw) [reply mern/client/src/Components/transcript-scroller.component.css:30]: Ok thanks!

**Comment by anishalatchman** [comment]
Scroll bar merged into integration in a seperate branch

**Pull request closed without merging**